host/discoverd.go

```diff
@@ -105,3 +109,15 @@ func (d *DiscoverdManager) ConnectPeer(ips []string) error {
 	}
 	return err
 }
+
+func (d *DiscoverdManager) UpdateTags(tags map[string]string) error {
+	d.mtx.Lock()
+	defer d.mtx.Unlock()
+	for k, v := range tags {
+		d.inst.Meta[host.TagPrefix+k] = v
```

Comment:
So I believe this boils down to how we document things, we should definitely have `flynn-host tags del` to delete a tag, so the only issue is what happens when you set a tag to the empty string. My implementation deletes the tag (which we can document) and yours will actually set it to empty (with the extra syntax to delete), but I really don't think we want empty tags right now, so I'm going to go with what is here.